manifests/a/Azul/Zulu/6/6.22.0.3/Azul.Zulu.6.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=AUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Azul.Zulu.6
+PackageVersion: 6.22.0.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- java
+FileExtensions:
+- class
+- jad
+- jar
+- java
+- jsp
+Installers:
+- Architecture: x64
+  InstallerUrl: https://cdn.azul.com/zulu/bin/zulu6.22.0.3-jdk6.0.119-win_x64.msi
+  InstallerSha256: FFF242EF3E734E2A4A3353F63796B86AF24A14098900A659BE5377D60BC9FF6B
```

```suggestion
  InstallerSha256: FFF242EF3E734E2A4A3353F63796B86AF24A14098900A659BE5377D60BC9FF6B
  ProductCode: '{A46C6FD9-1C02-43CB-8555-C36368B524BE}'
  AppsAndFeaturesEntries:
  - Publisher: Azul Systems, Inc.
    DisplayName: Zulu 6.22.0.3 (64-bit)
    DisplayVersion: 6.22.0.3
    InstallerType: wix
    ProductCode: '{A46C6FD9-1C02-43CB-8555-C36368B524BE}'
```
The ProductCode is important for matching since the DisplayName is actually just `Zulu`
The AppsAndFeaturesEntries will be used later